src/screens/WalletConnect/Home/WalletConnectHome.js

```diff
@@ -0,0 +1,230 @@
+import { type Chain, CHAIN } from 'models/Chain';
+import type { WalletConnectCmsApp } from 'models/WalletConnectCms';
+
+// Local
+import WalletConnectListItem from './components/WalletConnectListItem';
+import ConnectFloatingButton from './components/ConnectFloatingButton';
+import ConnectedAppsFloatingButton from './components/ConnectedAppsFloatingButton';
+import DeployOnEthereumBanner from './components/DeployOnEthereumBanner';
+
+function WalletConnectHome() {
+  const { t } = useTranslationWithPrefix('walletConnect.home');
+  const navigation = useNavigation();
+  const safeArea = useSafeAreaInsets();
+
+  const tabItems = useTabItems();
+  const [activeChain, setActiveChain] = React.useState<?Chain>(null);
+
+  const { numberOfColumns, columnWidth } = useColumnDimensions();
+  const { data: sections, isFetching } = useSectionData(activeChain, numberOfColumns);
+  const isDeployedOnEthereum = useIsActiveAccountDeployedOnEthereum();
+
+  const renderListHeader = () => {
+    const showDeployOnEthereumBanner = !isDeployedOnEthereum && activeChain === CHAIN.ETHEREUM;
+
+    return (
+      <ListHeader>
+        <Stories renderHeader={() => <SectionHeader>{t('headerStories')}</SectionHeader>} />
+
+        <TabBar items={tabItems} activeTab={activeChain} onActiveTabChange={setActiveChain} style={styles.tabs} />
+
+        {showDeployOnEthereumBanner && <DeployOnEthereumBanner style={styles.banner} />}
+      </ListHeader>
+    );
+  };
+
+  // Note: in order to achieve multicolumn layout, we group n normal items into one list row item.
+  const renderListRow = (items: WalletConnectCmsApp[]) => {
+    return (
+      <ListRow>{items.map(renderItem)}</ListRow>
+    );
+  };
+
+  const renderItem = (item: WalletConnectCmsApp) => {
+    const isEthereumOnly = activeChain === CHAIN.ETHEREUM || isEqual(item.chains.length, [CHAIN.ETHEREUM]);
+    const disabled = isEthereumOnly && !isDeployedOnEthereum;
+
+    return (
+      <WalletConnectListItem
+        key={item.id}
+        title={item.title}
+        iconUrl={item.iconUrl}
+        width={columnWidth}
+        disabled={disabled}
+      />
+    );
+  };
+
+  return (
+    <Container>
+      <HeaderBlock centerItems={[{ title: t('title') }]} navigation={navigation} noPaddingTop />
+
+      {!isFetching ? (
+        <SectionList
+          sections={sections ?? []}
+          renderSectionHeader={({ section }) => <SectionHeader>{section.title}</SectionHeader>}
+          renderSectionFooter={() => <SectionFooter />}
+          renderItem={({ item }) => renderListRow(item)}
+          keyExtractor={(items) => items[0]?.id}
+          ListHeaderComponent={renderListHeader()}
+          contentContainerStyle={{ paddingBottom: safeArea.bottom + FloatingButtons.SCROLL_VIEW_BOTTOM_INSET }}
+        />
+      ) : (
+        <>
+          {renderListHeader()}
+          <Center flex={1}>
+            <Spinner />
+          </Center>
+        </>
+      )}
```

Comment:
Maybe change this to:

{isFetching && ...}
{!isFetching && ...}

Totally optional tho, but just I think that ternary looks good only if it's single operator, but if there's more conditional rendering inside ternary then it can become a mess to read, wdyt?